hackathon_site/dashboard/frontend/src/slices/event/teamAdminSlice.ts

```diff
@@ -0,0 +1,100 @@
+import {
+    createAsyncThunk,
+    createEntityAdapter,
+    createSelector,
+    createSlice,
+} from "@reduxjs/toolkit";
+import { get } from "api/api";
+import { APIListResponse, Team } from "api/types";
+import { AppDispatch, RootState } from "slices/store";
+import { displaySnackbar } from "slices/ui/uiSlice";
+
+// TODO
+interface TeamAdminExtraState {
+    errorState: string | null;
+    isMoreLoading: boolean;
+    isLoading: boolean;
+}
+
+interface RejectValue {
+    status: number;
+    message: any;
+}
+
+const extraState: TeamAdminExtraState = {
+    errorState: null,
+    isLoading: false,
+    isMoreLoading: false,
+};
+
+const responseTime = 24;
```

Comment:
I'd suggest you make this constant all uppercase (`RESPONSE_TIME`) so ppl know it's fixed and you should export this constant too so you can use it in other files instead of having to declare it in every file.

Reply:
done